src/sdk.ts

```diff
@@ -807,6 +807,33 @@ export default class SDK {
     return this.client.setExtensionConfig<DataType, ResponseType>(this.identifier, config);
   }
 
+  /**
+   * Applies a set of patches to channel config state.
+   * This method requires a mapping of user_id to objects. The provided config
+   * objects per user are merged server-side. A key may be specified to be 'null'
+   * to delete the key on the server. Arrays are overwritten in their entirety on merge.
+   * Objects are merged recursively.
+   *
+   * This method can only set state for viewers who have shared their ID.
+   *
+   * This method requires an admin context.
+   * @async
+   * @param channelConfigs - a mapping of channelID to patch objects.
+   *
+   * @return {Promise} Will resolve on successful setting of config. Rejects on failure.
+   *
+   * @example
+   * sdk.patchChannelConfig({
+   *  '12452': { 'hello': 'world' },
+   *  '12422': { 'foo': 'bar' }
+   * });
```

Comment:
This function should not take multiple channels. It should only patch the channel for which the broadcaster is authorized.